src/domUpdates.js

```diff
@@ -8,29 +8,33 @@ const domUpdates = {
   },
 
   populateRoomArray(array, location) {
-    //console.log("trying")
-    //console.log(array)
-    location.innerHTML = "";
+    if (typeof array ===
```

Comment:
This is giving me some code smells... Instead of allowing for `array` to be a `string`, I would pass an empty array. So in Hotel.js I would delete: 
```
if (!roomsOnDate.length) {
      return `Sorry, there are no ${roomType}s available on that date. Please choose another room type or date.`
    }
```
And then here, change it to:
```
if (!array.length) {
  this.stringDisplay(location, "Sorry, no rooms found. Please choose another room type or date.")
}
```

Reply:
Great catch! Thank you, Nicole. Refactoring now

Reply:
@hannahhch